common-custom-user-data-gradle-plugin/src/main/java/com/gradle/CustomBuildScanEnhancements.java

```diff
@@ -8,6 +8,7 @@
 import org.gradle.api.invocation.Gradle;
 import org.gradle.api.tasks.testing.Test;
 
+import java.util.Optional;
```

Comment:
I think this was introduced in java 8. Are we at all worried about java 6 or 7 compat?

Reply:
Gradle 5 requires at least Java 8 to run, so I think this is ok.